Sources/SwiftDocC/Model/Rendering/References/ExternalLocationReference.swift

```diff
@@ -34,21 +43,26 @@ public struct ExternalLocationReference: RenderReference, URLReference {
 
     public init(identifier: RenderReferenceIdentifier) {
         self.identifier = identifier
+        self.url = identifier.identifier
     }
 
     public init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
 
         self.identifier = try container.decode(RenderReferenceIdentifier.self, forKey: .identifier)
+        let url = try container.decodeIfPresent(String.self, forKey: .url)
+        if let url = url {
+            self.url = url
+        } else {
+            self.url = self.identifier.identifier
+        }
```

Comment:
Is this actually optional? In the [spec it's not](https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json#L1823) so I think we can depend on it always being there.

Maybe we should just have this?

```suggestion
        self.url = try container.decode(String.self, forKey: .url)
```

Reply:
I was just being overly paranoid here. 😅 I can take it out, this shouldn't happen in practice.